### PR TITLE
taxonomy: improve category taxonomy

### PR DESCRIPTION
**Pull request by raphael0202**
Improvements of the category taxonomy, spotted when working on the category matching algorithm.

- en:Teas should be below en:Tea-based beverages instead of the more generic category en:Beverages
- en:Cereal pastas are also en:Pastas
- add italian translations

### CODE REVIEW

**Comment by alexgarel** [comment]
@raphael0202 FYI changed title because we have a specific prefix for taxonomy changes (so that they go in the right section in release please)

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -48379,6 +48379,7 @@ nl:Kokosboter, Kokosboters
 
 #do not put in nut butters, peanut is not a nut
 <en:Legume butters
+<en:Nut butters
```
Comment (alexgarel) [inline taxonomies/categories.txt:48382]: What about the comment just above ?

Comment (aleene) [reply taxonomies/categories.txt:48382]: Well peanut is not a nut. For allergens they differentiate between peanuts and tree nuts (hazel, wal, ...)

**Review by aleene — COMMENTED** [review]
The easter chocolate eggs can be different from chocolate eggs. Maybe introduce a parent for chocolate eggs?

File: taxonomies/categories.txt (hunk 1 above)
Comment (moon-rabbitOFF) [reply taxonomies/categories.txt:48382]: The distinction is also important for the calculation of the nutri-score I think

**Comment by aleene** [comment]
Need to add a test so that this can not be changed.

File: taxonomies/categories.txt (hunk 1 above)
Comment (raphael0202) [reply taxonomies/categories.txt:48382]: Indeed, I didn't see the comment. Many contributors make this mistake though, so it's quite frequent that "Nut butters" is suggested as a category for peanut butters.

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**